README.md

```diff
@@ -254,15 +268,95 @@ s := repo.Subscribers().All(func(e eventsourcing.Event) {
 s.Close()
 ```
 
-## Custom made event store
+## Snapshot
 
-If you want to store your events in another database beside the already implemented event stores: `sql`, `bbolt`, `esdb` or `memory`, you can implement a custom made event store. It has to implement the following  interface to support the eventsourcing.Repository.
+If an aggregate has alot of events it can take some time fetching it's event and building the aggregate. This can be optimised with the help of a snapshot. The snapshot is the state of the aggregate on a specific version. Instead of iterating all aggregate events only the events after the version is iterated and used to build the aggregate. The use of snapshots is optional and is exposed via the snapshot repository.
```

Comment:
```suggestion
If an aggregate has a lot of events it can take some time fetching it's event and building the aggregate. This can be optimized with the help of a snapshot. The snapshot is the state of the aggregate on a specific version. Instead of iterating all aggregate events, only the events after the version is iterated and used to build the aggregate. The use of snapshots is optional and is exposed via the snapshot repository.
```